lms/static/js/student_account/components/PasswordResetInput.jsx

```diff
@@ -0,0 +1,27 @@
+/* globals gettext */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { InputText } from '@edx/paragon/static';
+
+function PasswordResetInput(props) {
+  return (
+    <div className="form-field">
+      <InputText
+        id={props.name}
+        type="password"
+        themes={['danger']}
+        dangerIconDescription={gettext('Error: ')}
+        required
+        {...props}
+      />
+    </div>
+  );
+}
+
+PasswordResetInput.propTypes = {
+  name: PropTypes.string.isRequired,
```

Comment:
Was there a reason not to name this `id`? Too implementation details specific?

Reply:
The <input>'s id and name both get set to the same thing. So I wanted to pick just one of them to specify. Name felt more salient since it's a form input.